app/build.gradle

```diff
@@ -5,17 +5,23 @@ apply plugin: 'com.github.triplet.play'
 //apply plugin: 'io.fabric'
 
 android {
-    compileSdkVersion 24
-    buildToolsVersion '24.0.2'
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
 
     defaultConfig {
         applicationId 'uk.co.ribot.androidboilerplate'
         minSdkVersion 16
-        targetSdkVersion 24
+        targetSdkVersion 26
         testInstrumentationRunner "${applicationId}.runner.RxAndroidJUnitRunner"
         versionCode 1000
         // Major -> Millions, Minor -> Thousands, Bugfix -> Hundreds. E.g 1.3.72 == 1,003,072
         versionName '0.1.0'
+
+        javaCompileOptions {
+            annotationProcessorOptions {
+                includeCompileClasspath false
+            }
+        }
```

Comment:
I dont think this is needed post Butterknife 8.6?

See: https://github.com/JakeWharton/butterknife/issues/908